File: RepoTransBench/source_projects/Python/seatgeek_fuzzywuzzy/public_tests/test_public_fuzzywuzzy_pytest.py
import pytest
from fuzzywuzzy import process

def test_public_process_warning(caplog):
    """Check that a string reduced to 0 by processor logs a warning to stderr"""

    query = '......'
    choices = ['......']
    with caplog.at_level("WARNING"):
        _ = process.extractOne(query, choices)
    assert any(
        "Applied processor reduces input query to empty string" in m
        for m in caplog.messages
    )

File: RepoTransBench/target_projects/Python/C++/seatgeek_fuzzywuzzy/public_tests/test_public_fuzzywuzzy_pytest.cpp
#include <gtest/gtest.h>
#include "fuzzywuzzy/process.h"

TEST(PublicFuzzyWuzzyPytest, ProcessWarning) {
    // Simulate: query reduced to empty string, expect some warning mechanism (here, just assert returned value)
    std::string query = "......";
    std::vector<std::string> choices = {"......"};
    // Not possible to test logging directly in C++, so check the logic yields empty results or nullptr
    auto result = fuzzywuzzy::process::extractOne(query, choices);
    // We expect result to be empty or have a score of 0 if processor strips everything
    EXPECT_TRUE(result.first.empty() || result.second == 0);
}